ts/src/kuna.ts

```diff
@@ -760,24 +1006,62 @@ export default class kuna extends Exchange {
          * @name kuna#cancelOrder
          * @description cancels an open order
          * @param {string} id order id
-         * @param {string} symbol not used by kuna cancelOrder ()
+         * @param {string} symbol unified market symbol
          * @param {object} [params] extra parameters specific to the kuna api endpoint
          * @returns {object} An [order structure]{@link https://github.com/ccxt/ccxt/wiki/Manual#order-structure}
          */
         await this.loadMarkets ();
         const request = {
-            'id': id,
+            'orderId': id,
         };
-        const response = await this.privatePostOrderDelete (this.extend (request, params));
-        const order = this.parseOrder (response);
-        const status = order['status'];
-        if (status === 'closed' || status === 'canceled') {
-            throw new OrderNotFound (this.id + ' ' + this.json (order));
+        const response = await this.v4PrivatePostOrderPrivateCancel (this.extend (request, params));
+        //
+        //    {
+        //        "data": {
+        //            "success": true
+        //        }
+        //    }
+        //
+        const data = this.safeValue (response, 'data', {});
+        let market = undefined;
+        if (symbol !== undefined) {
+            market = this.market (symbol);
         }
-        return order;
+        return this.parseOrder (data, market);
+    }
+
+    async cancelOrders (ids: string[], symbol: string = undefined, params = {}) {
+        /**
+         * @method
+         * @name kuna#cancelOrder
+         * @description cancels an open order
+         * @param {string} ids order ids
+         * @param {string} symbol not used by kuna cancelOrder
+         * @param {object} [params] extra parameters specific to the kuna api endpoint
+         * @returns {object} An [order structure]{@link https://github.com/ccxt/ccxt/wiki/Manual#order-structure}
+         */
+        await this.loadMarkets ();
+        const request = {
+            'orderIds': ids,
+        };
+        const response = await this.v4PrivatePostOrderPrivateCancelMulti (this.extend (request, params));
+        //
+        //    {
+        //        "data": [
+        //            {
+        //                "id": "c7fc5b2b-bd9d-48c1-a458-a83412669fe2",   // Unique identifier of a canceled order
+        //                "success": true                                 // Status for this order
+        //            },
+        //            ...
+        //        ]
+        //    }
+        //
+        const data = this.safeValue (response, 'data', []);
+        return this.parseOrders (data);
     }
 
     parseOrderStatus (status) {
+        // TODO
```

Comment:
we're missing some states here namely Pending
![image](https://github.com/ccxt/ccxt/assets/43336371/560ed6be-51b1-44e5-b52e-1d9676d3f61e)